lib/http-bridge/test/bench/Main.hs

```diff
@@ -322,3 +313,16 @@ getCurrentSlot net = calcSlot <$> startTime net <*> getPOSIXTime
 
 logQuiet :: SlotId -> SlotId -> IO ()
 logQuiet _ _ = pure ()
+
+-- | Initialize logging at the specified minimum 'Severity' level.
+initTracer
+    :: Severity
+    -> Text
+    -> IO (CM.Configuration, Switchboard Text, Trace IO Text)
+initTracer minSeverity cmd = do
+    c <- defaultConfigStdout
+    CM.setMinSeverity c minSeverity
+    CM.setSetupBackends c [CM.KatipBK, CM.AggregationBK]
+    (tr0, sb) <- setupTrace_ c "bench cardano-http-bridge:restore"
```

Comment:
Copied the setup from the CLI. Maybe we're fine with just a `setupTrace_ ` and none of the other settings here.

Reply:
```suggestion
c <- defaultConfigStdout
tr <- setupTrace (Right c) "cardano-wallet-http-bridge"
return (tr `appendName` "bench" `appendName` "restore")
``` 

Though... to be honest, I don't think `waitForWalletSync` needs any tracing function... It doesn't actually provides us with much more than what the restoration workers already do.